js/index.js

```diff
@@ -19,15 +19,64 @@ for (let index = 0; index < skills.length; index++) {
 
     skill.innerText = skills[index];
     skillsList.appendChild(skill)
+}
+console.log(skills)
 
+const messageForm = document.querySelector("form[name=leave_message]")
+console.log(messageForm);
 
-}
+messageForm.addEventListener('submit', event => {
+
+
+    const name = event.target.name.value
+
+    //console.log(Name)
+    const emailAddress = event.target.email.value
+    //console.log(emailAddress)
+    const message = event.target.message.value
+    console.log(name, emailAddress, message);
+
+    event.preventDefault();
```

Comment:
this is a bit of a nitpick but it's good practice to always put this right at the top of your `addEventListener` callback to ensure that no side-effects like refreshing the page can happen before the rest of your code runs

```js
messageForm.addEventListener('submit', event => {
   event.preventDefault();

    const name = event.target.name.value
etc...
```

Reply:
I made changes in my index.html and index.js files. i added and pushed them. should i merge? or make another pull request?